api_helpers.py

```diff
@@ -9,8 +9,9 @@
 from hs_restclient import HydroShare, HydroShareAuthBasic
 
 from file_ops import extract_fileinfo_from_url, retry_func
-from settings import logger, headers, HYDROSHARE_VERISON
+from settings import logger, headers, HYDROSHARE_VERISON, README_COLUMN_MAP, README_FILENAME
 from utils_logging import log_exception
```

Comment:
You can keep the column map path in settings, so it can be called, but the file object itself should live in data and get imported from there

Reply:
@cuahsimarko 
I am going to merge this PR leaving you to refactor it. No much changes to submit this week from my side as I was review migration logs and testing different things in a separate file.